packages/qwik/src/core/shared/error/error.ts

```diff
@@ -50,7 +50,6 @@ export const codeToText = (code: number, ...parts: any[]): string => {
       "Element must have 'q:container' attribute.", // 42
       'Unknown vnode type {{0}}.', // 43
       'Materialize error: missing element: {{0}} {{1}} {{2}}', // 44
-      'SsrError: {{0}}', // 45
```

Comment:
You can't just delete a line here, each error must be on a single line so that error links work.

If you renumber all errors it's ok but once it's stable you shouldn't renumber any more. What you can do is use old numbers for new errors I suppose.

Reply:
yeah I know, it is alpha thats why I removed the error